system-tests/test/config_modules_spec.ts

```diff
@@ -0,0 +1,37 @@
+import systemTests from '../lib/system-tests'
+
+// All projects live inside of projects/config-cjs-and-esm
+describe('cypress config with esm and cjs', function () {
+  systemTests.setup()
+
+  ;[
+    'config-cjs-and-esm/config-with-mjs',
+    'config-cjs-and-esm/config-with-cjs',
+    'config-cjs-and-esm/config-with-js-module',
+    'config-cjs-and-esm/config-with-ts-module',
+
+    // This covers Vite and SvelteKit e2e projects
+    'config-cjs-and-esm/config-with-ts-module-and-esbuild',
+  ].forEach((project) => {
+    systemTests.it(`supports modules and cjs in ${project}`, {
+      project,
+      testingType: 'e2e',
+      spec: 'app.cy.js',
+      browser: 'chrome',
+      expectedExitCode: 0,
+    })
+  })
+
+  ;[
+    'config-cjs-and-esm/config-with-ts-module-component',
```

Comment:
I wonder if we need a `forEach` for an array with one entry? Seems redundant.

Reply:
Parallelism and it'll be nicer if we want to add more CT examples. I don't see a problem with it.